theano/sandbox/rng_mrg.py

```diff
@@ -865,7 +865,10 @@ def uniform(self, size, low=0.0, high=1.0, ndim=None, dtype=None,
             nstreams = self.n_streams(size)
         rstates = self.get_substream_rstates(nstreams, dtype)
 
-        node_rstate = shared(rstates)
+        d = {}
+        if kwargs.has_key('target'):
+            d = dict(target=kwargs.pop('target'))
```

Comment:
THis can be simplified by:

    d = kwargs.pop('target', {})

the last argument in the default value if target isn't in the dic.t

Reply:
If kwargs has 'target', pop() will only return the value. But d should be a dict like {'targat':value}.

Reply:
Right. What I had in mind won't work, so there is just the small changes from my other comment.

thanks